Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+pipeline {
+    agent any
+    options {
+        disableConcurrentBuilds()
+    }
+    environment {
+        DOCKER_REPO = "docker.dev.pureport.com"
+        DEB_BUILDER_IMAGE = "pureport/deb-builder"
+        UID = getUid()
+        GID = getGid()
+    }
+    stages {
+        stage('Build') {
+            when {
+                branch "pureport/*"
+            }
+            steps {
+                script {
+                    def workspace = pwd()
+                    sh "git fetch --tags"
```

Comment:
What's the `git fetch --tags` for?

Reply:
Apparently the Jenkins Git plugin doesn't fetch the tags when checking out the repo, and they're required by the build scripts.